src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  transition: transform $global-transition-time ease;
```

Comment:
The transition property should only be applied to properties that change on hover to improve performance. In this case, it seems you're only changing the `transform` property on hover, so you should specify `transition: transform ...;` instead of using the shorthand that applies to all properties.